packages/flutter_test/lib/src/window.dart

```diff
@@ -151,6 +151,13 @@ class TestWindow implements Window {
     _window.onLocaleChanged = callback;
   }
 
+  @override
+  String get initialLifecycleState => _initialLifecycleStateTestValue ?? 'AppLifecycleState.resumed';
```

Comment:
`resumed` seems like it probably should not be the default....any reason that this is the right thing to start with?

Reply:
also, don't you check for this value being null elsewhere in this PR?  Can that condition every possibly happen given this default?

Reply:
This was just meant to be a valid value for the test window. Since the test window is never reflecting the state of a real app, it just needs to be any valid value.

Reply:
Ok.  But the behavior of the real app can have a null value here, right?  If so, then would it be better to force tests to explicitly set this if they want a non-null value?  Otherwise, your test window reflects an implicit behavior that doesn't take place in reality, right?

Reply:
Ahh that makes sense, will do!